desktop/wasm/index.md

```diff
@@ -5,28 +5,39 @@ keywords: Docker, WebAssembly, wasm, containerd, engine
 toc_max: 3
 ---
 
-Wasm (short for WebAssembly) is a faster, lighter alternative to the Linux & Windows containers you’re using in Docker today (with [some tradeoffs](https://www.docker.com/blog/docker-wasm-technical-preview/){:target="_blank" rel="noopener" class="_"}). 
+Wasm (short for WebAssembly) is a faster, lighter alternative to the Linux &
```

Comment:
I think we preferred the language "fast, light alternative" rather than faster/lighter